underscore.js

```diff
@@ -73,6 +73,7 @@
       };
       // The 2-parameter case has been omitted only because no current consumers
       // made use of it.
+      case void 0:
```

Comment:
This is an unnecessary change because there is already a null check a few lines above this:

https://github.com/jashkenas/underscore/blob/5d8ab5e37c9724f6f1181c5f95d0020815e4cb77/modules/index.js#L60